app/src/main/java/com/anytypeio/anytype/ui/sharing/SharingFragment.kt

```diff
@@ -56,6 +75,14 @@ class SharingFragment : BaseBottomSheetComposeFragment() {
                         when(option) {
                             SAVE_AS_BOOKMARK -> vm.onCreateBookmark(url = sharedData.data)
                             SAVE_AS_NOTE -> vm.onCreateNote(sharedData.data)
+                            SAVE_AS_IMAGE -> vm.onShareMedia(listOf(sharedData.data))
+                            SAVE_AS_FILE -> vm.onShareMedia(listOf(sharedData.data))
+                            SAVE_AS_IMAGES -> vm.onShareMedia(
+                                uris = (sharedData as SharingData.Images).uris
```

Comment:
Is it possible not to use unsafe operator "as" here?

Reply:
fixed